test/conv_common.hpp

```diff
@@ -1646,6 +1660,36 @@ struct conv_driver : test_driver
 
     void run()
     {
+        if(input.desc.GetSize() != in_layout.size() ||
+           weights.desc.GetSize() != fil_layout.size() || input.desc.GetSize() != out_layout.size())
+        {
+            MIOPEN_LOG_E("layout not match dimension size!");
```

Comment:
This looks like a test error, shall we `std::cerr << "... FAILED ... " ... ` and return?

Reply:
OK, change to std::cerr...